drivers/storage/ec2/ec2.go

```diff
@@ -76,10 +88,16 @@ func (d *driver) Init(r *core.RexRay) error {
 		return goof.WithFields(ef(), "error getting instance id doc")
 	}
 
-	auth := aws.Auth{
-		AccessKey: d.accessKey(),
-		SecretKey: d.secretKey(),
+	auth, err := aws.GetAuth(
```

Comment:
when using IAM for credentials I think a temporary token is issued. I'm only guessing, but based on the "owning" func name here, `Init`, don't we need to periodically refresh the temporary token that AWS issues when using IAM credentials?